### ISSUE TITLE
Some docs incorrect v1.18 kubectl run command

### ISSUE THREAD

**Issue by iaoiui** [kubernetes/website#20782]
**This is a Bug Report**
kubectl run with --replicas option has been deleted at following PR.
https://github.com/kubernetes/kubernetes/pull/87077/files

But, there are some sample commands which use the removed option in docs.

```
$find ./website/content/en/docs -type f -name "*.md" -print0 | xargs -0 grep -e "kubectl run" | grep  "replicas" 
./website/content/en/docs/tasks/administer-cluster/namespaces-walkthrough.md:kubectl run cattle --image=k8s.gcr.io/serve_hostname --replicas=5
./website/content/en/docs/tasks/administer-cluster/namespaces.md:    kubectl run snowflake --image=k8s.gcr.io/serve_hostname --replicas=2 -n=development
./website/content/en/docs/tasks/administer-cluster/namespaces.md:    kubectl run cattle --image=k8s.gcr.io/serve_hostname --replicas=5 -n=production
./website/content/en/docs/concepts/policy/resource-quotas.md:kubectl run nginx --image=nginx --replicas=2 --namespace=myspace
./website/content/en/docs/reference/kubectl/overview.md:`run`        | `kubectl run NAME --image=image [--env="key=value"] [--port=port] [--replicas=replicas] [--dry-run=server|client|none] [--overrides=inline-json] [flags]` | Run a specified image on the cluster.
```

**Problem:**
Some document is misleading about kubectl run command.

**Proposed Solution:**
Update some docs.
- Use "kubectl create deploy" instead of "kubectl run" 
- add "kubectl scale deploy" on the next line

**Page to Update:**
https://kubernetes.io/docs/tasks/administer-cluster/namespaces
https://kubernetes.io/docs/tasks/administer-cluster/namespaces-walkthrough
https://kubernetes.io/docs/concepts/policy/resource-quotas

**Kubernetes Version**
```
# kubectl version
Client Version: version.Info{Major:"1", Minor:"18", GitVersion:"v1.18.2", GitCommit:"52c56ce7a8272c798dbc29846288d7cd9fbae032", GitTreeState:"clean", BuildDate:"2020-04-16T11:56:40Z", GoVersion:"go1.13.9", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"18", GitVersion:"v1.18.2", GitCommit:"52c56ce7a8272c798dbc29846288d7cd9fbae032", GitTreeState:"clean", BuildDate:"2020-04-30T20:19:45Z", GoVersion:"go1.13.9", Compiler:"gc", Platform:"linux/amd64"}
```

**Comment by iaoiui** [comment]
I created PR #20784

**Comment by sftim** [comment]
In a review for the PR, I recommended `kubectl <verb> deployment` instead of  `kubectl <verb> deploy`.

**Comment by tengqm** [comment]
Issue fixed by #20784.
/close
